Makefile

```diff
@@ -14,21 +15,29 @@ openwrt/feeds.conf:
 	cd ../dist/openwrt; \
 	cp ../../build/feeds.conf .; \
 	./scripts/feeds update -a; \
-	./scripts/feeds install -a;
+	./scripts/feeds install -a; \
+	cd feeds/packages; patch -p1 < ../../../../build/0001-glib2-make-libiconv-dependent-on-ICONV_FULL-variable.patch; \
```

Comment:
Actually it was needed to fix bluez package build issue too...so until we get fix from openwrt packages, we can have this patch to fix those issues